### ISSUE TITLE
Coalesce macro not picking up __fl fields

### ISSUE THREAD

**Issue by PowdyPowPow** [dbt-labs/stitch-utils#8]
### Issue description
I have several tables synced by Stitch in BigQuery that contain 2 columns split for datatype: `time`and `time__fl`
When using the macro referencing a source `{{ stitch_utils.coalesce_fields(relation = source('dataset','table')) }}` it is doesn't seem to be picking up `__fl` columns and not doing the `coalesce`

### Results
The generated query doesn't include the coalesce of the two split columns.

### System information
```
Running with dbt=0.13.1
Installing fishtown-analytics/stitch_utils@=0.1.4
  Installed from version 0.1.4

Installing fishtown-analytics/dbt_utils@=0.1.24
  Installed from version 0.1.24
```

The operating system you're running on:
OSX 10.13.6

The python version you're using (probably the output of `python --version`)
stable 3.7.3

### Possible Solution
I was able to get this working by overriding the macro and including 'fl' in the list here:
```
            {%- set col_split = col.column.split('__') -%}
            {%- if col_split|length > 1 and col_split[-1]|lower in (
                'bl','st','it','bigint','string','double','boolean'
            ) -%}
```

**Comment by drewbanin** [comment]
Thanks for the report @PowdyPowPow! Your solution looks like a good one - are you interested in making a PR with that change? We'd be happy to merge it :)

**Comment by PowdyPowPow** [comment]
happy to @drewbanin. I just tried pushing a branch but got permissions denied error?

**Comment by drewbanin** [comment]
Thanks! You're going to want to fork this repo, then push your changes to your fork. When you open a PR in your repo, you should be presented with an option to make the PR against `fishtown-analytics/stitch-utils`.

This happened because GitHub doesn't allow users to push to a repo that they are not a member of. Let me know if you have any trouble getting this set up!

**Comment by PowdyPowPow** [comment]
@drewbanin thanks, i think i got it now. please have a look when you get a chance and let me know if there is anything i should adjust.

**Comment by drewbanin** [comment]
Thanks @PowdyPowPow - I'll check it out today!

**Comment by drewbanin** [comment]
Fixed in https://github.com/fishtown-analytics/stitch-utils/pull/9 and released in [0.1.5](https://hub.getdbt.com/fishtown-analytics/stitch_utils/latest/)

Thanks @PowdyPowPow!
